### DISCUSSION TITLE
[updated] New scrum team for autonomous employee in slack

### DISCUSSION THREAD

**Discussion by AxGucci** [daveshap/ACE_Framework#37] [General]
Hello :)
Axel (PO) is writing to you.

### **SCRUM TEAM** 
_(👤 = Available / ✅ = Covered)_

- Product Owner ✅
- Scrum Master 👤
- Tech Lead 👤
- Developers 👤👤 _(let's start with 2 devs; if we are solid, we can bring in more people)_

- **Use case**: enterprise (an assistant for small businesses (ideally startups) that significantly enhances teamwork, organization, planning, communication, etc.) in SLACK.
_* The use case is based on my personal background, what I know best. It's open to changes/suggestions. The important thing is to get started._

### **How to Join the Team?**
Send me a message on LinkedIn: https://www.linkedin.com/in/axel-gualda/

If you don't have LinkedIn, send me a message on WhatsApp: +54 9 11 6631-7681.
Why?
We need skin in the game and accountability, so I'm taking ownership of this team and the product.
I need to know who I'm going to work with, not just a nickname. I don't have time to waste.
You need to show your face, full name, and truly commit.

### **The Team**

**Axel - Product Owner**

Hello! This is Axel, from Argentina. Second-time founder (edtech with +14,000 students and +25 team members).
Industrial Engineer and economics lecturer at the university.
My best role would be as PO (product owner), although I can act as an interim scrum master until someone with better knowledge and more experience comes along.
My main skill is execution, not so much organization. Concrete plans, conscious decisions, short deadlines, and significant progress.
Intermediate knowledge of AI, conceptually much better than technically.
As I said, my skill is execution. I don't like theory, but application. I don't like talking, but doing.
So, write to me on LinkedIn and we'll update this discussion with new additions.

**Comment by daveshap** [comment]
I hope you get more interest soon!

**Reply by AxGucci** [reply]
Me too :')

**Comment by AxGucci** [comment]
This post originally had a different approach, but since it didn't start, I've updated it to make it more accessible

**Comment by hexadecimal-REBOOT** [comment]
"Hello all, I'm thrilled to share that our team is actively planning the next steps for this MVP. We're creating an inclusive environment where all skill sets and perspectives are valued. Whether you're a developer, have experience in scrum, or are new to enterprise solutions, your input could be invaluable. Diversity of thought can only make this project stronger, so please don't hesitate to reach out and get involved. Let's collaborate to make something extraordinary!"

Scrum Master
